### PR TITLE
update information about container networking metrics

### PR DESCRIPTION
**Pull request by ameowlia**
* this feature is only available in OS currently

Questions for @pspinrad: 
* I used the term "TPCF" in this PR. Is that the correct way to refer to the TAS tile? 
* Can you please cherry-pick this for all supported versions of this doc?

### CODE REVIEW

**Review by anita-flegg — CHANGES_REQUESTED** [review]

File: container-metrics.html.md.erb [hunk 1]
```diff
@@ -71,12 +71,30 @@ The following table describes all Diego container metrics:
...
     <td><code>rx_bytes</code></td>
-    <td>Received network traffic in bytes for an app instance.</td>
+    <td>
+    Received network traffic in bytes for an app instance.
+    <% if vars.platform_code = 'CF' %>
+      Operators can enable this metric by setting garden-runc-release's
+      <code>garden.enable_container_network_metrics</code> bosh
+      property to <code>true</code>.
+    <% else %>
+      This metric is not available in TPCF yet.
```
[the remark this answers is not in the document]
Comment (ameowlia) [reply container-metrics.html.md.erb:81]: Thank you for these suggestions @anita-flegg! I will update this.

[the remark this answers is not in the document]
Comment (ameowlia) [reply container-metrics.html.md.erb:81]:
```suggestion
      This metric is not available in Tanzu Platform for Cloud Foundry.
```

[the remark this answers is not in the document]
Comment (ameowlia) [reply container-metrics.html.md.erb]: Fixed both cases!

**Comment by ameowlia** [comment]
@anita-flegg - I have made the changes you suggested. Thank you for your help!

**Comment by anita-flegg** [comment]
Thanks, @ameowlia !
@pspinrad to cherry-pick to all the places this should go -- I assume TAS-6.0, TAS-5.0, TAS-4.0, and the new 10.0 branch...?

**Comment by ameowlia** [comment]
Yes! All of those places are correct!

**Pull request merged**